files/image_config/caclmgrd/caclmgrd

```diff
@@ -202,6 +213,36 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
                                                      (docker_mgmt_ip, self.namespace_mgmt_ip))
         return allow_internal_docker_ip_cmds
 
+    def generate_translate_front_panel_snmp_traffic_commands(self, namespace):
```

Comment:
map_snmp_traffic_from_namespace_to_host. instead of front_panel we should use namespace.

Reply:
updated the naming everywhere to fwd_snmp_traffic_from_namespace_to_host